web/frontend/src/api/navigation.ts

```diff
@@ -75,8 +75,7 @@ export const formatObstacles = (list: GeoObstacle[]): Obstacle[] => {
       geometries: obstacle.geometriesList.map((geometry) => {
         const { center } = geometry;
         const pose = new ViamObject3D();
-        const th = THREE.MathUtils.degToRad(center?.theta ?? 0);
-        pose.orientationVector.set(center?.oX, center?.oY, center?.oZ, th);
+        pose.orientationVector.set(center?.oX, center?.oY, center?.oZ, center?.theta ?? 0);
```

Comment:
Could you explain this fix? 

ViamObject3Ds are by default radians, and to my understanding, Viam prefers degrees.

Reply:
This makes sure that obstacles on the RC card are rendered correctly.
If I create an obstacle with theta 90, this then is rendered with theta 1.57 on the rc card. 
Which makes sense since 1.57 is 90 in radians.
However, the obstacle itself appears to be rotated 1.57 **degrees**.
This fix removes us translating degrees into radians.

> ViamObject3Ds are by default radians, and to my understanding, Viam prefers degrees.

I agree that Viam prefers degress, but I didn't know that `ViamObject3Ds` are be default in radians.
If this is true, then I suspect my fix maybe isnt the right approach.. Do you think something in prime or app might be responsible for the disconnect? i.e something isn't translating the 1.57 back to 90 down the line..

Reply:
yes, of course :)